godot-core/src/builtin/projection.rs

```diff
@@ -337,8 +369,12 @@ impl Projection {
     /// has the given horizontal field of view (in degrees) and aspect ratio.
     ///
     /// _Godot equivalent: Projection.get_fovy()_
-    pub fn fovy_of(fov_x: real, aspect: real) -> real {
-        real::from_f64(InnerProjection::get_fovy(fov_x.as_f64(), aspect.as_f64()))
+    pub fn fovy(fov_x: real, aspect: real) -> real {
+        let half_angle_fov_x = f64::to_radians(fov_x.as_f64() * 0.5);
+        let vertical_transform = f64::atan(aspect.as_f64() * f64::tan(half_angle_fov_x));
+        let full_angle_fov_y = f64::to_degrees(vertical_transform * 2.0);
+
+        real::from_f64(full_angle_fov_y)
```

Comment:
We named the method `fovy_of` in the initial implementation, see https://github.com/godot-rust/gdext/pull/124#discussion_r1119248900.

But I think `create_fovy` might actually be a better name, in line with the other constructors. Godot's `get_fovy` for a static method is terrible naming.

Can you name it `create_fovy` and add a `#[doc(alias)]` with the Godot name?

Reply:
Added `#[doc(alias = "get_fovy")]`, and changed the function name to `create_fovy` :)